pkg/apis/messaging/v1beta1/kafka_channel_types_test.go

```diff
@@ -47,3 +49,54 @@ func TestKafkaChannelGetStatus(t *testing.T) {
 		t.Errorf("GetStatus did not retrieve status. Got=%v Want=%v", config.GetStatus(), status)
 	}
 }
+
+func TestKafkaChannelSpec_ParseRetentionDuration(t *testing.T) {
```

Comment:
nit: let's not the `_` - let's do camelCase

Reply:
Yeah, assuming that was just IntelliJ - sorry - will fix!